/aruco_node:
    ros__parameters:
        # Aruco detection parameters
        marker_size: 0.065 # Size of the markers in meters
        aruco_dictionary_id: DICT_7X7_250 # Aruco dictionary type
        image_topic: /camera/color/image_raw # Input image topic
        camera_info_topic: /camera/color/camera_info # Input camera info topic with camera intrinsic parameters
        camera_frame: camera_depth_optical_frame # Camera link frame of reference
        # Output topics
        detected_markers_topic: /aruco_markers # Output topic with detected markers (aruco poses + ids)
        markers_visualization_topic: /aruco_poses # Output topic with visualization of detected markers as pose array
        display_image_topic: /aruco_image # Output topic with visualization of detected markers drawn on image
